### DISCUSSION TITLE
!pip install openpnm==3.0.0

### DISCUSSION THREAD

**Discussion by engphb** [PMEAL/OpenPNM#2600] [Q&A] [answered]
Hello professor. I'm using google colab and would like to test OpenPNM version 3.0. However, when I try to use the command:

!pip install openpnm==3.0.0

The following error message appears:

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
ERROR: Could not find a version that satisfies the requirement openpnm==3.0.0 (from versions: 1.1, 1.2.0, 1.3.0, 1.4.0, 1.4.5, 1.4.6, 1.5.0, 1.6.1, 1.6.2, 1.7.0, 2.0.0b2, 2.0.0b3, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0, 2.2.0, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.4.0, 2.4.1, 2.4.2, 2.5.0, 2.5.1, 2.6.0, 2.6.1, 2.7.0, 2.8.1, 2.8.2)
ERROR: No matching distribution found for openpnm==3.0.0

This error does not occur with I digit :
!pip install openpnm==2.8.2

**Comment by jgostick** [comment, marked as answer]
My first guess is that Google uses a mirrored/cloned version of Pypi that is a bit behind the actual index.  If you go to the pypi site you can see that 3.0.0 is there: https://pypi.org/project/openpnm/.  I see that the logger message says it looks in pypi.org/simple.  Not sure what the /simple means, but that must be the difference.  My suggestion is to stick with 2.8 for now, and wait it out.
